RecoTauTag/RecoTau/plugins/RecoTauCleaner.cc

```diff
@@ -180,13 +180,13 @@ namespace
 	const reco::RecoTauPiZero& piZero = signalPiZeroCandidates.at(iPiZero);
 	std::cout << " piZero #" << iPiZero << ": Pt = " << piZero.pt() << ", eta = " << piZero.eta() << ", phi = " << piZero.phi() << ", mass = " << piZero.mass() << std::endl;
       }
-      const std::vector<reco::PFCandidatePtr>& isolationPFCands = tauRef_->isolationPFCands();
-      size_t numPFCands = isolationPFCands.size();
-      std::cout << "isolationPFCands = " << numPFCands << std::endl;
-      for ( size_t iPFCand = 0; iPFCand < numPFCands; ++iPFCand ) {
-	const reco::PFCandidatePtr& pfCand = isolationPFCands.at(iPFCand);
-	std::cout << " pfCand #" << iPFCand << " (" << pfCand.id() << ":" << pfCand.key() << "):" 
-		  << " Pt = " << pfCand->pt() << ", eta = " << pfCand->eta() << ", phi = " << pfCand->phi() << std::endl;
+      const auto& isolationCands = tauRef_->isolationCands();
```

Comment:
@perrotta @slava77 @steggema after merging this PR there is a list of massive failures in the clang build in this class:

```
Begin processing the 14th record. Run 1, Event 14, LumiSection 1 on stream 2 at 07-Apr-2019 10:29:06.605 CEST


A fatal system signal has occurred: segmentation violation
The following is the call stack containing the origin of the signal.

Sun Apr  7 10:29:06 CEST 2019

Current Modules:

Module: RecoTauCleaner:hpsPFTauProducerSansRefs (crashed)
Module: PoolOutputModule:AODSIMoutput
Module: RPCDigiProducer:simMuonRPCDigis
Module: FastSimProducer:fastSimProducer

A fatal system signal has occurred: segmentation violation
```